src/all/holonometria/src/eu/kanade/tachiyomi/extension/all/holonometria/Holonometria.kt

```diff
@@ -112,13 +112,16 @@ class Holonometria(
         return chapters
     }
 
-    override fun chapterListSelector() = "#Archive article"
+    override fun chapterListSelector() = ".manga-detail__list-item"
 
     override fun chapterFromElement(element: Element) = SChapter.create().apply {
         setUrlWithoutDomain(element.selectFirst("a")!!.attr("href"))
-        name = element.select(".ttl").text()
-        date_upload = element.selectFirst(".data--date")?.text().parseDate()
-        scanlator = element.selectFirst(".data--category")?.text()
+        val nameText = element.select(".manga-detail__list-title").text()
+        name = nameText.split("【", "】")[1]
+        date_upload = element.selectFirst(".manga-detail__list-date")?.text().parseDate()
+        scanlator = "COVER Corporation"
+        val number = name.replace("-", ".").split(" ")[1].toFloatOrNull()
+        chapter_number = number ?: 0f
```

Comment:
app will most likely extract the number correctly, if not then you can put your own logic